resources/asciidoctor/lib/copy_images/extension.rb

```diff
@@ -54,8 +55,8 @@ def process_callout(block)
       coids = block.attr('coids')
       return unless coids
 
-      coids.scan(/CO(?:\d+)-(\d+)/) do
-        @copier.copy_image block, "images/icons/callouts/#{$1}.#{callout_extension}"
+      coids.scan(CALLOUT_RX) do |(index)|
```

Comment:
There's no need to wrap `index` in`()`

Reply:
I wrapped it so I could get the index rather than a single element array containing the index.

Reply:
ah, makes sense.